docs/accessing-orcd/ssh-login.md

```diff
@@ -0,0 +1,103 @@
+---
+tags:
+ - Logging in with SSH
+---
+
+# Logging in with SSH via Terminal
+
+You can log into our systems via SSH through your local terminal. Using SSH in a terminal or command line window on your desktop is the traditional way to access HPC Systems. This method offers the most flexibility, allowing you to start interactive and batch jobs to run your code, download data, and install packages.
+
+## Terminal by Operating System
+A terminal window is a window with a command line interface. 
+
+To log into our systems, we use the terminal to SSH into the system. SSH (Secure Shell) is the primary way to log into remote systems. Once you initiate the SSH command, the shell in your terminal will no longer run on your computer but on the remote system. Authentication is required, either using a password or SSH keys. To set up SSH keys, please refer to insert url.
```

Comment:
Needs link to the SSH Keys page :)

Reply:
I added the SSH Key Setup Page (https://orcd-docs.mit.edu/accessing-orcd/ssh-setup/). The URL is not currently available since that page is a part of this pull request, but it should work correctly once it's merged.